README.adoc

```diff
@@ -35,12 +35,14 @@ constraints.
 Bean Validation is a Java specification that simplifies data validation and error 
 checking. Bean validation uses a standard way to validate data stored in 
 JavaBeans. Validation can be performed manually or with integration with other 
-specifications and frameworks, such as CDI, JPA, or JSF. To set rules on data, 
-apply constraints by using annotations or XML configuration files. Bean
-validation provides both built-in constraints and the ability to create custom 
-constraints. Bean validation allows for validation of both JavaBean fields and 
-methods. For method-level validation, both the input parameters and return value 
-can be validated.
+specifications and frameworks, such as Contexts and Dependency Injection (CDI), 
+Java Persistence API (JPA), or JavaServer Faces (JSF). To learn more about CDI, 
```

Comment:
Kind of don't want to add the CDI link here, as there're no links for the other 2 specs, JPA and JSF. IMO, we don't need the link to CDI guide in the intro. You're just mentioning about these specs and frameworks, it can be expected that user research to understand more if they'd like. If you're really describing more CDI content, and it's related to the guide, then it can be linked.